src/controller/proxy/controller.go

```diff
@@ -122,6 +124,7 @@ func (c *controller) ProxyManifest(ctx context.Context, p *models.Project, art l
 	}
 	// Push manifest in background
 	go func() {
+		ctx := orm.NewContext(ctx, borm.NewOrm())
```

Comment:
The first parm should be context.Background?

Reply:
updated